### PR TITLE
A few template changes.

### PR DESCRIPTION
**Pull request by dbuenzli**

### CODE REVIEW

File: share/templates/default/module.xml [hunk 1]
```diff
@@ -3,25 +3,22 @@
...
-
       <t:insert name=".decl-body" with=".decl"/>
     </div>
   </t:let>
 
   <t:let name="module.rhs">
     <t:if exists=".sig">: <t:insert name="module.rhs-sig" with=".sig"/></t:if>
-    <t:if exists=".alias">= <t:insert name="nav.path" with=".alias.path"/></t:if>
+    <t:if exists=".alias">= <t:insert name="nav.path" with=".alias.path"/>
+    </t:if>
```
Comment (dsheets) [inline share/templates/default/module.xml]: need to elide space

**Comment by dsheets** [comment]
The corresponding CSS would be nice. Waiting a while to see if it shows up before I merge.

**Comment by dbuenzli** [comment]
I have removed the web fonts and rely on standard fonts now. If you want I can get some CSS in but bear in mind that it's really not finished and very messy.

**Comment by dbuenzli** [comment]
Ok so I added a WIP stylesheet.

**Comment by dsheets** [comment]
Great! Much better than the current one so even in its incomplete state, it is worthwhile to merge.

Thanks!

**Pull request merged**